lib/types/gpt.nix

```diff
@@ -108,6 +146,7 @@ in
     _create = diskoLib.mkCreateOption {
       inherit config options;
       default = ''
+        sgdisk -Z ${config.device}
```

Comment:
isn't our wipefs already doing something like that?

Reply:
we should probably call sgdisk -Z then in the disk_deactivate script instead of here